load/UpdateDBPTextFilesets.py

```diff
@@ -84,14 +84,24 @@ def invokeBiblePublisher(self, inp, fullFilesetPath):
 
 		return None
 
-	def invokeSofriaCli(self, fullFilesetPath, filesetLptsName):
+	def invokeSofriaCli(self, fullFilesetPath, inputFileset):
 		# invoke sofria
 		# this is the invocation of the sofria-cli. It should be similar to the invocation of BiblePublisher
```

Comment:
remove comment line 89

Reply:
@bradflood it has been removed as you can see in the following line:

https://github.com/faithcomesbyhearing/dbp-etl/pull/91/files#diff-38516f84a6c11ce60e5c2399f71fb8e10c8fffbc654c8e51082685269a35aa11R63